### PR TITLE
Switched to using FTPServer.jl for testing

### PR DESCRIPTION
**Pull request by rofinn**

### CODE REVIEW

**Review by nicoleepp — APPROVED** [review]
LGTM

**Comment by nicoleepp** [comment]
Linux Travis doesn't seem to like it though

**Comment by rofinn** [comment]
Looks like I needed to set `PYTHON=""` in the travis file.

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ julia:
   - nightly
 env:
   - CONDA_JL_VERSION="2"
```
Comment (iamed2) [inline .travis.yml]: Still mandatory?

**Review by iamed2 — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (rofinn) [reply .travis.yml]: Not, really. I can change it now if you like, but I wanted to make sure I haven't broken the tests on 2 first and I don't want to run the FTPClient tests on both 2 and 3. That's what I'm doing in FTPServer.jl.

File: test/runtests.jl [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import Compat: Sys, occursin, read
 using Compat.Test
 using FTPClient
+using FTPServer
+using FTPServer: username, password, hostname, port, HOMEDIR, tempfile
```
Comment (omus) [inline test/runtests.jl:5]: Requires v0.1.1

File: test/REQUIRE [hunk 3]
```diff
@@ -1,2 +1,3 @@
 Conda
```
Comment (omus) [inline test/REQUIRE]: Drop this now?

**Review by omus — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (rofinn) [reply .travis.yml]: Turns out I was testing on py27 locally anyways, so I just dropped it now :p

**Pull request merged**